evap/staff/tests/test_importers.py

```diff
@@ -571,16 +627,16 @@ def test_import_new_contributor(self):
         success_messages, __, __ = PersonImporter.process_source_evaluation(
             ImportType.CONTRIBUTOR, self.evaluation1, test_run=True, source_evaluation=self.evaluation2
         )
-        self.assertIn("1 contributors would be added to the evaluation", "".join(success_messages))
-        self.assertIn(f"{self.contributor2.email}", "".join(success_messages))
+        self.assertIn("1 contributor would be added to the evaluation", "".join(success_messages))
+        self.assertIn(str(self.contributor2.email), "".join(success_messages))
```

Comment:
what is the purpose of this change? Is this an accidental merge-undo?

Reply:
I changed it back. I think the str was a mistake